### PR TITLE
Add automatic text wrapping for markdown files

### PR DESCRIPTION
**Pull request by gregth**

### CODE REVIEW

File: .vimrc [hunk 1]
```diff
@@ -179,3 +179,6 @@ set foldlevel=99
 " Split behavior
 set splitbelow
 set splitright
+
+" Markdown-specific settings
+autocmd FileType markdown set formatoptions+=t | set tw=79`
```
Comment (dionyziz) [inline .vimrc]: What is the backtick for?

Comment (gregth) [inline .vimrc]: Oops, it's unneeded, I will remove it.

**Comment by dionyziz** [comment]
squash

**Comment by gregth** [comment]
Squashed.

**Pull request merged**